requirements/default.txt

```diff
@@ -147,3 +145,6 @@ https://github.com/mathjazz/silme/archive/v0.9.4.zip#egg=silme==0.9.4 \
 
 https://github.com/mathjazz/translate/archive/2.2.5.zip#egg=translate-toolkit==2.2.5 \
     --hash=sha256:9a23bda0f3ec07f36b6b662bf2a25492b6a354709500c736213229e94f462443
+
+https://github.com/mathjazz/python-caighdean/archive/v0.0.4.zip#egg=caighdean==0.0.4 \
```

Comment:
Any idea how we can make sure we don't forget to switch back to the official package as soon as a new version is released?

Reply:
I'll get an email notification from GitHub when it's merged? :)

Reply:
![](https://i.giphy.com/media/3dkXFcZgXEu9noZQCE/giphy.webp)